apps/nextjs/e2e/development/beforeSend.spec.ts

```diff
@@ -28,6 +33,6 @@ test.describe('beforeSend', () => {
 
     await page.waitForLoadState('networkidle');
 
-    expect(messages).toHaveLength(5);
+    expect(messages).toHaveLength(6);
```

Comment:
**suggestion:** since we already had an issue with test assertion being too weak, you could start asserting the content of these messages